### PR TITLE
Add page on Brick/REC modeling with 223

### PR DESCRIPTION
**Pull request by gtfierro**

### CODE REVIEW

File: guides/brick-rec.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+# Using 223 with Brick and RealEstateCore
+
+The [Brick](https://brickschema.org) and [RealEstateCore](https://realestatecore.io) ontologies can be used to augment 223 models with a richer vocabulary of concepts.
+There are a few reasons to consider doing this:
```
Comment (MatthewSteen) [inline guides/brick-rec.md:4]:
```suggestion
The [Brick](https://brickschema.org) and [RealEstateCore](https://realestatecore.io) ontologies can be used to augment 223 models with a richer vocabulary of concepts, which offer the following benefits.

File: guides/brick-rec.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+```
+
+By stating that the `bldg:air-temp` entity is also a `brick:Supply_Air_Temperature_Sensor`, Brick-based applications can more easily find the data they are looking for.
+223 requires multiple annotations be combined in order to model this concept.
+
+The Brick class also clearly states the role of the entity.
+The use of the QUDT annotations and Brick types means that this 223 model is also a valid Brick model!
+
+Using Brick can simplify some queries against the model.
+Without Brick, finding the `bldg:air-temp` entity would require this query:
```
Comment (MatthewSteen) [inline guides/brick-rec.md:54]:
```suggestion
For example, finding the `bldg:air-temp` entity without Brick would require this query:
```

File: guides/brick-rec.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+SELECT ?damper ?temp WHERE {
+    ?damper a s223:Damper ;
+        s223:hasProperty ?temp.
+    ?temp a s223:QuantifiableObservableProperty ;
+        qudt:hasQuantityKind qudtqk:Temperature ;
+        s223:hasAspect s223:Role-Supply .
+}
+```
+
+whereas with Brick, the query can be simplified to
```
Comment (MatthewSteen) [inline guides/brick-rec.md:70]:
```suggestion
With Brick, the query above can be simplified to the following where the `brick:hasPoint` relationship is inferred automatically.
```

File: guides/brick-rec.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+PREFIX brick: <https://brickschema.org/schema/Brick#>
+
+SELECT ?damper ?temp WHERE {
+    ?damper a s223:Damper ;
+        brick:hasPoint ?temp .
+    ?temp a brick:Supply_Air_Temperature_Sensor .
+}
+```
+
+The `brick:hasPoint` relationship is inferred automatically.
```
Comment (MatthewSteen) [inline guides/brick-rec.md:84]:
```suggestion
```

File: guides/brick-rec.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+## Using Brick `Equipment`s with 223
+
+Brick's extensive `Equipment` classes can also be used with 223.
+Any 223 `Equipment` can also be annotated with a Brick `Equipment` class, for example:
```
Comment (MatthewSteen) [inline guides/brick-rec.md:89]:
```suggestion
Any 223 `Equipment` can also be annotated with a Brick `Equipment` class as shown in the example below. 
```

**Review by MatthewSteen — COMMENTED** [review]
Looks good, just a few suggestions. Do you think we should add external links to explore.open223.info and REC?

**Comment by gtfierro** [comment]
Thanks for the edits!

**Pull request merged**